apps/posts/src/views/post-analytics/components/Newsletter.tsx

```
@@ -0,0 +1,104 @@
+import OpenedList from './newsletter/OpenedList';
+import React from 'react';
+import SentList from './newsletter/SentList';
+import {Badge} from '@tryghost/shade';
+import {StatsTabItem, StatsTabTitle, StatsTabValue, StatsTabs, StatsTabsGroup} from './StatsTabs';
+
+interface newsletterProps {};
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)